src/compas/datastructures/mesh/operations/split.py

```diff
@@ -193,7 +193,7 @@ def mesh_split_face(mesh, fkey, u, v):
     >>> mesh = Mesh.from_obj(compas.get("faces.obj"))
     >>> fkey = mesh.get_any_face()
```

Comment:
technically, i guess this should become `face = mesh.face_sample(size=1)[0]`

Reply:
also, to make this example more robust we should exclude faces on the boundary

Reply:
i guess the sample functions should be updated with an `exclude` parameter.
```python
face = mesh.face_sample(size=1, exclude=mesh.faces_on_boundary())[0]
```

Reply:
👍